modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,232 @@
+            dept: engr <2>
+        podSelector: {}
+  priority: 55
+  subject: <3>
+    namespaces:
+      matchExpressions:
+      - key: security
+        operator: In
+        values: <4>
+        - restricted
+        - confidential
+        - internal
+----
+<1> Specifies a node or set of nodes in the cluster using the `matchExpressions` field. It will select all nodes if the label is present, but left empty.
+<2> Specifies all the pods labeled with `dept: engr`.
+<3> Specifies the subject of the ANP which includes any namespaces that match the labels used by the network policy. The example matches any of the namespaces with `restricted`, `confidential`, or `internal` level of `security`.
+<4> Specifies key/value pairs for `matchExpressions` field.
+====
+
+[id="egress-traffic-control-networks-peer-external-destinations_{context}"]
+== Using networks peer to control egress traffic towards external destinations
+
+Cluster administrators can use CIDR ranges for `networks` peer and apply an egress traffic policy to those nodes.
+
+The following example uses `networks` peer and combines ANP and BANP policies to isolate egress traffic to a range of specified CIDR ranges.
+
+.Example of ANP and BANP using `networks` peers
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: network-as-egress-peer
+spec:
+  priority: 70
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-egress-to-external-dns-servers"
+    action: "Deny"
+    to:
+    - networks:
+      - 8.8.8.8/32
+      - 8.8.4.4/32
+      - 208.67.222.222/32
+    - pods:
+        namespaceSelector: {}
+        podSelector: {}
+  - name: "allow-all-egress-to-intranet"
+    action: "Allow"
+    to:
+    - networks:
+      - 172.30.0.10/32 <1>
+    - pods:
+        namespaceSelector: {}
+        podSelector: {}
+    ports:
+      - portNumber:
+          protocol: UDP
+          port: 53
+  - name: "allow-all-intra-cluster-traffic"
+    action: "Allow"
+    to:
+    - networks:
+      - 10.128.0.0/14
+      - 172.30.0.0/16 <2>
+    - pods:
+        namespaceSelector: {}
+        podSelector: {}
+  - name: "pass-all-egress-to-internet"
+    action: "Pass"
+    to:
+    - networks:
+      - 0.0.0.0/0 <3>
+---
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  name: default
+spec:
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-all-egress-to-internet"
+    action: "Deny"
+    to:
+    - networks:
+      - 0.0.0.0/0 <3>
+---
+----
+<1> Use `networks` to specify a range of CIDR networks outside of the cluster.
+<2> Specifies the CIDR ranges for the intra-cluster traffic from your resources.
+<3> Specifies a strong `Deny` egress policy to the internet by setting `networks` values to - 0.0.0.0/0. Make sure you have an allow policy at a higher priority before setting a `Deny` to 0.0.0.0/0 because this will deny all traffic including to API servers.
```

Comment:
I'd make this a red like with EFW but I leave it to you, 
```suggestion
<3> Specifies a `Deny` egress  to everything by setting `networks` values to - 0.0.0.0/0. Make sure you have higher priority allow rules to necessary destinations before setting a `Deny` to 0.0.0.0/0 because this will deny all traffic including to Kubernetes API and DNS servers.
```